examples/example_add_adiabat.py

```diff
@@ -0,0 +1,113 @@
+# %% [markdown]
+"""
+# example_add_adiabat
+
+This example demonstrates how to read in terra output in NetCDF format
+and add a theoretical adiabat to the temperatures.
+"""
+
+# %% [markdown]
+"""
+Let's import all the necessary python objects.
+"""
+
+# %%
+from pathlib import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import glob
+from terratools.terra_model import read_netcdf
+from terratools.example_data import example_terra_model
+from terratools.terra_model import _calculate_adiabat
+
+# %% [markdown]
+"""
+Now lets download and read in the example mantle convection model
+netcdf file and print some summary information.
+"""
+
+# %%
+# Download and cache model
+path = example_terra_model()
+
+# read in the model
+model = read_netcdf(glob.glob(path))
```

Comment:
This can just be:

```python
model = read_netcdf([path])
```

No need for `glob`, including no need to import it.

(We should allow a single file name as a string as well as an iterable of strings; for another PR.)

It would be good to do this for the others as well.